Source/SwiftLintFramework/Rules/CyclomaticComplexityRule.swift

```diff
@@ -82,19 +81,19 @@ public struct CyclomaticComplexityRule: ASTRule, ConfigurationProviderRule {
 
             return complexity +
                 (complexityStatements.contains(statementKind) ? 1 : 0) +
-                measureComplexity(file, dictionary: subDict)
+                measureComplexity(in: file, dictionary: subDict)
         }
 
         if hasSwitchStatements {
-            return reduceSwitchComplexity(complexity, file: file, dictionary: dictionary)
+            return reduceSwitchComplexity(initial: complexity, file: file, dictionary: dictionary)
         }
 
         return complexity
     }
 
     // Switch complexity is reduced by `fallthrough` cases
 
-    private func reduceSwitchComplexity(_ complexity: Int, file: File,
+    private func reduceSwitchComplexity(initial complexity: Int, file: File,
```

Comment:
I debated that, but went with this since the method name is `reduceSwitchComplexity`, so you can kind of infer that "initial" is referring to the initial complexity, but now that you bring it up, it's obvious this could be clearer. I'll rename the label to `initialComplexity`. 👍